src/components/common/RolloverLogsDialog.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <app-dialog
+    v-model="open"
+    :title="$t('app.general.title.rollover_logs')"
+    :save-button-text="$t('app.general.btn.accept')"
+    :max-width="450"
+    @save="sendAccept"
+  >
+    <v-card-text>
+      <v-radio-group v-model="application">
```

Comment:
![image](https://user-images.githubusercontent.com/25269274/222795376-06179627-1aa7-4112-9d77-2b1c11b9a8cf.png)
```suggestion
      <v-radio-group
        v-model="application"
        class="mt-0"
        hide-details
      >
```

Reply:
I've also made the dialog width a bit smaller (450 -> 400) and updated the main screenshot on this PR.